cirq-web/cirq_ts/README.md

```diff
@@ -0,0 +1,96 @@
+## Cirq Typescript Development
+
+*This directory contains the code and instructions for visualization tools in a web browser or Colab/Juptyer notebooks. We do this using Typescript. While necessary, this is only half of the code needed to run the `cirq-web` project. For information on how to integrate projects here with Python and the wider Cirq package, see the `cirq_web` directory.*
+
+Visualizations run on [NodeJS](https://nodejs.org/en/), and we use [npm](https://www.npmjs.com/) for package management. To start developing, clone the repository and run `npm install` within this directory to install the necessary packages and begin development. You will need to install Node and npm if you haven't already.
+
+
+For developing 3D visualizations, we rely on the [three.js](https://threejs.org/) framework. Understanding their documentation is critical to understanding 3D visualizations in Cirq. 
```

Comment:
```suggestion
For developing 3D visualizations, we rely on the [three.js](https://threejs.org/) framework. 
```
This sentence is unnecessary - people can prioritize their level of involvement themselves in the threejs docs, our job is to point them to the places that they can use.